### DISCUSSION TITLE
Remove accidentally faces group

### DISCUSSION THREAD

**Discussion by martadinata666** [matiasdelellis/facerecognition#596] [Q&A] [answered]
So i accidentally, group this man X to man Y, how do i revert it? how do i need revert it? Or there is command to drop a whole group? thanks.

**Comment by matiasdelellis** [comment, marked as answer]
Hi @martadinata666 
When you enter the view of the images of the person, at the end there is a button to see the groups of faces. Use that to find the wrong group and rename it again. 😉
